src/encoded/static/data/privacy-policy.md

```diff
@@ -17,11 +17,14 @@ We may collect non-personal information about the computer, mobile device or oth
 
 ## Cookies
 
-Cookies are files with small amount of data, which may include an anonymous unique identifier. Cookies are sent to your browser from a web site and stored on your computer's hard drive. We use cookies to understand and save your preferences for future visits and compile aggregate data about site traffic and site interaction so that we can offer better site experiences and tools in the future. We may contract with third-party service providers to assist us in better understanding our site visitors. You can instruct your browser to refuse all cookies or to indicate when a cookie is being sent. However, if you do not accept cookies, you may not be able to use some portions of our Portal or sign in.
+Cookies are files with small amount of data, which may include an anonymous unique identifier. Cookies are sent to your browser from a web site and stored on your computer's hard drive. We use cookies to understand and save your preferences for future visits and compile aggregate data about site traffic and site interaction so that we can offer better site experiences and tools in the future. You can instruct your browser to refuse all cookies or to indicate when a cookie is being sent. This can be performed in your browser's settings or by installing an ad-blocker. However, if you do not accept cookies, you may not be able to use some portions of our Portal or sign in.
```

Comment:
Of curiosity, are we writing this text or are we getting it from some legal office within HMS/Harvard?

Reply:
Originally we gathered text of the doc from various Harvard sites, primarily harvard.edu, hms.harvard.edu, and dbmi.hms.harvard.edu. And kind of combined them together meaningfully to make a superset (there might thus be some redundancy, i.e. 2 sections talking about cookies).

For these edits, wrote text self based on some keywords/verbage in Google Analytics documentation.

Reply:
I have no idea how to get in contact with HMS legal people (or who they might be) :|.